webserver/__init__.py

```diff
@@ -19,6 +25,9 @@ def create_app():
     import config
     app.config.from_object(config)
 
+    # Run the scheduled jobs
+    scheduledJobs = ScheduledJobs(app.config)
```

Comment:
I'm saying when I run `create_app` function, I don't expect it to do anything else apart from things directly related to creating an instance of Flask app.